inscriptions/ISic003323.xml

```diff
@@ -199,12 +199,12 @@
             <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
                     <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/><persName type="attested"><name type="nomen"><seg part="F"><unclear>i</unclear><unclear>o</unclear></seg></name> <name type="cognomen"><supplied reason="lost">I</supplied><unclear>n</unclear><unclear>n</unclear><unclear>o</unclear><unclear>c</unclear>enti</name></persName>
```

Comment:
When you have a cluster of unclear characters, you can tag them together within one <unclear>.